hathor/transaction/block.py

```diff
@@ -143,7 +143,7 @@ def get_block_parent(self) -> 'Block':
         assert isinstance(block_parent, Block)
         return block_parent
 
-    def get_funds_fields_from_struct(self, buf: bytes) -> bytes:
+    def get_funds_fields_from_struct(self, buf: bytes, *, verbose: bool = False) -> bytes:
```

Comment:
We should use this verbose in `TxOutput.create_from_bytes`, right?

And also log `self.version` and `outputs_len`

Reply:
Fixed. Thanks!